src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseContentFiles.cs

```diff
@@ -26,7 +23,7 @@ public GivenThatWeWantToUseContentFiles(ITestOutputHelper log) : base(log)
         [Fact]
         public void It_handles_content_files_correctly()
         {
-            const string targetFramework = "netcoreapp2.0";
+            const string targetFramework = "netcoreapp3.1";
```

Comment:
Does this issue repro with the 6.0? If so, we prefer to use the latest TFMs in our tests.